### DISCUSSION TITLE
Weird Regression in .net 8 Maui Project: "Unable to find package Microsoft.AspNetCore.App.Runtime.win-arm"

### DISCUSSION THREAD

**Discussion by dgerding** [dotnet/maui#16485] [Q&A]
Hi All,
Using the hot of the presses preview. Can't get around a build error that cropped up in the last steps of migrating a .net 7 maui solution to .net preview latest.

NU1102	Unable to find package Microsoft.AspNetCore.App.Runtime.win-arm with version 8.0.0-preview.6.23329.11

I've manually uninstalled and reinstalled the Preview SDK and done same with Visual Studio. Currently _only have_ 2023 17.7.6 installed and still no luck. 

Any ideas or just work arounds appreciated.

Dave G

**Comment by dgerding** [comment]
Just a follow up that no matter what I try, I can't seem to get around the following build error on this machine. I've removed all VS and .net sdk instances, restarted a couple of times and then installed today's (preview 6). 

---csproj : error NU1102: Unable to find package Microsoft.AspNetCore.App.Runtime.win-arm with version (= 8.0.0-preview.6.23329.11)
---csproj : error NU1102:   - Found 125 version(s) in nuget.org [ Nearest version: 8.0.0-preview.4.23260.4 ]
---csproj : error NU1102:   - Found 0 version(s) in Microsoft Visual Studio Offline Packages
---csproj : error NU1102:   - Found 0 version(s) in C:\Program Files\dotnet\library-packs
---csproj : error NU1101: Unable to find package Microsoft.NETCore.App.Runtime.Mono.win-arm64. No packages exist with this id in source(s): C:\Program Files\dotnet\library-packs, Microsoft Visual Studio Offline Packages, nuget.org

**Reply by MartyIX** [reply]
It might help to include sources from https://github.com/dotnet/maui/blob/main/NuGet.config. Just once to get the dependencies. It helped me once. Maybe it will help, maybe not.

**Reply by dgerding** [reply]
Thanks, @MartyIX, that worked for one build and then failed again. Found a different solution, see below.

**Comment by dgerding** [comment]
Hi All,
It turns out that the problem was an old explicit package include in the csproj file for 

    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230628000-preview1" />

Removing the explicit package reference for Microsoft.WindowsAppSDK in csproj fixes the problem, presumably by permitting upstream dependencies to now "auto include" this reference as needed.

Hope this helps someone else.
